chat.iamb.iamb.metainfo.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>chat.iamb.iamb</id>
+
+  <name>iamb</name>
+  <summary>A terminal Matrix client for Vim addicts</summary>
+  <url type="homepage">https://iamb.chat</url>
+
+  <releases>
+    <release version="0.0.9" date="2024-03-28"/>
+  </releases>
+
+  <developer_name>Ulyssa</developer_name>
+  <update_contact>git@ulyssa.dev</update_contact>
+
+  <metadata_license>CC-BY-SA-4.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://iamb.chat/static/images/iamb-demo.gif</image>
+      <caption>Example conversation within iamb</caption>
+    </screenshot>
+  </screenshots>
+
+  <description>
+    <p>
+      iamb is a client for the Matrix communication protocol.  It provides a terminal user interface with familiar Vim keybindings, and includes support for multiple profiles, threads, spaces, notifications, reactions, custom keybindings, and more.
+    </p>
+  </description>
+
+  <icon type="stock">chat.iamb.iamb</icon>
+  <launchable type="desktop-id">chat.iamb.iamb.desktop</launchable>
+
```

Comment:
Missing content_rating, make sure to validate this file using https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation

Reply:
I've added a content rating to this file in ulyssa/iamb#315:

```
  <content_rating type="oars-1.1">
    <content_attribute id="social-chat">intense</content_attribute>
  </content_rating>
```

I followed the questions at [the OARS generator](https://hughsie.github.io/oars/generate.html) linked to from [here](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#oars-information).